web/public/mgmt/config.yml

```diff
@@ -3757,6 +3757,18 @@ collections:
                     name: "ctaButton"
                     widget: string
                     required: true
+              - label: "Seo"
+                name: "seo"
+                widget: object
```

Comment:
Is the `required` key not needed for the `SEO` label?

Reply:
It's not. This is an object comprised of multiple fields. The individual fields can be required, but the overall container doesn't have an associated input.

Here's an example of what something similar looks like:
<img width="774" alt="Screenshot 2024-09-09 at 3 04 05 PM" src="https://github.com/user-attachments/assets/aec83fb8-2f06-4922-b1db-dfc4fec00785">